### DISCUSSION TITLE
What is this device menu that appear when I use an hotkey ?

### DISCUSSION THREAD

**Discussion by Belphemur** [Belphemur/SoundSwitch#731] [FAQ]
Since updating to v6.0.0, I get this menu popping when I use a hotkey.

![image](https://user-images.githubusercontent.com/197810/131253927-d775690a-1f98-49f7-b072-8a980daf28cc.png)

**Comment by Belphemur** [comment]
This is the Quick Menu.

The idea is to reproduce the same feature that Windows has for language: 
![image](https://user-images.githubusercontent.com/197810/131253958-bdbec672-632d-4711-8a14-94b372f5f152.png)

When you use one of your hotkey (playback or recording), the menu will show with the current default audio endpoint.
By using the hotkey again, you'll see the selection change.

Moreover, you can also click on any item in the menu to directly change to the wanted device.

## Disable it
To disable it, go into the Settings > Quick Menu > Uncheck

![image](https://user-images.githubusercontent.com/197810/131254046-47efa9d7-6799-41df-98e9-6dfff079b9ac.png)

**Comment by sander1095** [comment]
Thanks for being able to turn it off.

Perhaps a better idea for future features would be to have them off by default but after the update introduce the user to them?

For me, they appear on the diagonally left of the middle of the screen, which was very jarring :)

But still, thanks for producing such an awesome piece of software

**Reply by Belphemur** [reply]
Hello,

That a good point, to have some kind of tutorial explaining the new feature.
